### PR TITLE
debian folders bringup for one-runtime pkg

### PR DESCRIPTION
**Pull request by chunseoklee**
## Purpose
To enable one-runtime debian pkg for launchpad.net(public)
Sorry for dirty patches since 
1) it is hard to enable this on public(launchpad) and 
2) we need to support both one-compiler and one-runtime in a single debian/rules file(Please help!)

## Issue 
- For internel  one-compiler pkg(on CI), we need to pass 'BUID_COMPILER=1 and CI_BUILD=1' envvar on build. 
- one-runtime is not tested on internal CI
- one-compiler pkg will be generated on launchpad.net w/o 'BUID_COMPILER=1'

### CODE REVIEW

**Comment by chunseoklee** [comment]
cc @mhs4670go

**Comment by chunseoklee** [comment]
Via offline discussion with @mhs4670go, Here is the conclusion.

1) remove the existing debian folder
2) create two (debian-like) folders debian_runtime/debian_compiler in infra/debian/ folder.

**Comment by chunseoklee** [comment]
@nnfw-bot help

**Comment by chunseoklee** [comment]
@nnfw-bot test all

**Comment by chunseoklee** [comment]
build and test are done on launchpad.net(See https://github.com/Samsung/ONE/issues/7191)

File: infra/debian/runtime/rules [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/make -f
+export VERSION=1.16.0
```
Comment (mhs4670go) [inline infra/debian/runtime/rules:2]: You could get version like this.
```
DEBVER := $(shell dpkg-parsechangelog -SVersion)
```

Comment (chunseoklee) [reply infra/debian/runtime/rules]: it works!

File: infra/debian/runtime/rules [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	install -d debian/tmp/usr/lib/pkgconfig
+	install Product/out/lib/* debian/tmp/usr/lib/
+	install -d debian/tmp/usr/include/
+	cp -r Product/out/include/* debian/tmp/usr/include/
+	sed -i 's:@libdir@:\/usr\/lib:g' ./packaging/nnfw.pc.in
+	sed -i 's:@includedir@:\/usr\/include:g' ./packaging/nnfw.pc.in
+	sed -i 's:@version@:${DEBVER}:g' ./packaging/nnfw.pc.in
+	install -m 0644 packaging/nnfw.pc.in -T debian/tmp/usr/lib/pkgconfig/nnfw.pc
+	dh_install -p nnfw
+	dh_install -p nnfw-dev
```
Comment (mhs4670go) [inline infra/debian/runtime/rules:25]:
```suggestion
	dh_install
```
I think this is enough?

**Comment by chunseoklee** [comment]
@mhs4670go @hseok-oh PTAL
hope that bab8a0e545b0a04faad5bba7f5e491b021657748 is the last commit in the PR. I have confirmed that deb(https://code.launchpad.net/~one-runtime/+archive/ubuntu/ppa/+build/21781771) works fine.

**Comment by hseok-oh** [comment]
@chunseoklee Can't we share `*.tar.gz` in `/packaging` ?

**Comment by chunseoklee** [comment]
> @chunseoklee Can't we share `*.tar.gz` in `/packaging` ?

Possible. I will check it.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**